### PR TITLE
Fix apollo dbsnapshot test

### PR DESCRIPTION
**Pull request by Efrat1**
Fixed test_restore_from_snapshot_of_other

### CODE REVIEW

File: tests/apollo/test_skvbc_dbsnapshot.py [hunk 1]
```diff
@@ -965,6 +960,19 @@ async def wait_for_created_snapshots_metric(self, bft_network, replicas, expecte
                 else:
                     break
 
+    async def wait_for_fast_paths_metric_to_update(self, bft_network, replica_id, old_num_of_fast_paths):
```
Comment (toly-kournik) [inline tests/apollo/test_skvbc_dbsnapshot.py:963]: Don't we have an existing method to check that replica has caught up?

Comment (Efrat1) [reply tests/apollo/test_skvbc_dbsnapshot.py]: I used wait_for_consensus_path with path_type OPTIMISTIC_FAST.
Let me know if you thought about something else.

Comment (toly-kournik) [reply tests/apollo/test_skvbc_dbsnapshot.py]: I don't know why whis method was defined in test_skvbc_dbsnapshot.py, when it's a bft network functionality.
There's wait_for_fast_path_to_be_prevalent() defined in bft

**Review by toly-kournik — APPROVED** [review]
LGTM! thanks Efrat

**Pull request merged**